Virtualization/Mesos/6.0/zabbix-mesos-via-http.xml

```diff
@@ -0,0 +1,7657 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<zabbix_export>
+    <version>6.4</version>
```

Comment:
Please change the directory according to the template version.

Reply:
Done